### PR TITLE
Port changeDetector to OpenLiberty

### PR DESCRIPTION
**Pull request by aguibert**
#build

### CODE REVIEW

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
build is good. Ready for merging once code review is approved.

#run-libby-bot

File: dev/build.changeDetector/test-resources/3581.diff [hunk 1]
```diff
@@ -0,0 +1,181 @@
+dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mongodb-2.0-ssl.feature
```
Comment (KyleAure) [inline dev/build.changeDetector/test-resources/3581.diff:1]: Is this diff file supposed to be checked in?

**Review by KyleAure — APPROVED** [review]

File: dev/build.changeDetector/test-resources/3581.diff (hunk 1 above)
Comment (aguibert) [reply dev/build.changeDetector/test-resources/3581.diff:1]: These were used for unit tests I believe

**Pull request merged**